hive-runtime/NOTICE

```diff
@@ -42,11 +42,11 @@ NOTICE file:
 |     * Redistributions of source code must retain the above copyright
 | notice, this list of conditions and the following disclaimer.
 |     * Redistributions in binary form must reproduce the above
-| copyright notice, this list of conditions and the following disclaimer
-| in the documentation and/or other materials provided with the
+| copyright notice, this list of conditions, and the following disclaimer
+| In the documentation and/or other materials provided with the
 | distribution.
 |     * Neither the name of Google Inc. nor the names of its
-| contributors may be used to endorse or promote products derived from
+| Contributors may be used to endorse or promote products derived from
```

Comment:
```suggestion
| contributors may be used to endorse or promote products derived from
```
I think it would be better to leave this part as it is rather than start a new sentence.